homeassistant/components/flux_led/__init__.py

```diff
@@ -1 +1,201 @@
-"""The flux_led component."""
+"""The Flux LED/MagicLight integration."""
+from __future__ import annotations
+
+from datetime import timedelta
+import logging
+from typing import Any
+
+from flux_led import BulbScanner, WifiLedBulb
+
+from homeassistant import config_entries
+from homeassistant.components.light import DOMAIN as LIGHT_DOMAIN
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    ATTR_MODE,
+    CONF_DEVICES,
+    CONF_HOST,
+    CONF_MAC,
+    CONF_MODE,
+    CONF_NAME,
+    CONF_PLATFORM,
+    CONF_PROTOCOL,
+    EVENT_HOMEASSISTANT_STARTED,
+)
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.debounce import Debouncer
+from homeassistant.helpers.event import async_track_time_interval
+from homeassistant.helpers.typing import ConfigType
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import (
+    CONF_COLORS,
+    CONF_CUSTOM_EFFECT,
+    CONF_CUSTOM_EFFECT_COLORS,
+    CONF_CUSTOM_EFFECT_SPEED_PCT,
+    CONF_CUSTOM_EFFECT_TRANSITION,
+    CONF_SPEED_PCT,
+    CONF_TRANSITION,
+    DEFAULT_EFFECT_SPEED,
+    DISCOVER_SCAN_TIMEOUT,
+    DOMAIN,
+    FLUX_HOST,
+    FLUX_LED_EXCEPTIONS,
+    MODE_AUTO,
+    STARTUP_SCAN_TIMEOUT,
+    TRANSITION_GRADUAL,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS = ["light"]
+DISCOVERY_INTERVAL = timedelta(minutes=15)
+REQUEST_REFRESH_DELAY = 0.35
```

Comment:
```suggestion
REQUEST_REFRESH_DELAY = 0.5
```
These appear to be slow to change state sometimes

Reply:
sometimes it takes event longer so maybe not worth it since it self corrects every 5s